packages/api/src/command/medical/patient/query-init.ts

```diff
@@ -38,14 +38,16 @@ export const storeQueryInit = async ({ id, cxId, cmd }: StoreQueryParams): Promi
       transaction,
     });
 
-    return patient.update(
-      {
-        data: {
-          ...patient.data,
-          ...cmd,
+    return (
+      await patient.update(
+        {
+          data: {
+            ...patient.data,
+            ...cmd,
+          },
         },
-      },
-      { transaction }
-    );
+        { transaction }
+      )
+    ).dataValues;
```

Comment:
this presumably worked before ... 

what is the addition of .dataValues doing here and how is the function return type not changing?

Reply:
`.dataValues` is a property of the Sequelize model that has the actual values. When we get an object from the DB using sequelize, it has a bunch of other stuff that allows one to use it as an Active Record - not out pattern.

It's same shape as the interface.

The reason for this is so we minimize the risk to send non-metriport data over the wire to the lambda, since the result of this is passed as a paramter to `getConsolidatedPatientDataAsync`.

If you search for `.dataValues` you'll see we use it in multiple places.

Reply:
Discussed in huddle